platform/mros2-platform.cpp

```diff
@@ -23,17 +23,29 @@
 
 #include "cmsis_os.h"
 #include "wifi.h"
-
+#include "ethernet_example_main.h"
 
```

Comment:
I think this code block can be modified as the below.

```
#ifdef CONFIG_MROS2_ESP32_NETIF_WIFI
#include "wifi.h"
#elif CONFIG_MROS2_ESP32_NETIF_ETH_SPI
#include "ethernet_example_main.h"
#else
```

I'm not sure this modification worked well, so I'm happy if you could check the operation.